contracts/Staking.sol

```diff
@@ -7,14 +7,15 @@ contract Staking {
 
     // Parameters
     uint128 public constant ValidatorThreshold = 1 ether;
-    uint32 public constant MinimumRequiredNumValidators = 4;
 
     // Properties
     address[] public _validators;
     mapping(address => bool) _addressToIsValidator;
     mapping(address => uint256) _addressToStakedAmount;
     mapping(address => uint256) _addressToValidatorIndex;
     uint256 _stakedAmount;
+    uint32 _maximumNumValidators;
+    uint32 _minimumNumValidators;
```

Comment:
You've made these values private, but haven't provided adequate getter methods.

Please consider providing them to improve the UX 🙏 

cc @Kourin1996

Reply:
Added in https://github.com/0xPolygon/staking-contracts/pull/10/commits/7cca3f7a5e005edabca976a999dd89e013c56ddf